spec/automation/unit/method_validation/filter_by_dialog_parameters_spec.rb

```diff
@@ -49,7 +49,7 @@ def root_service_template_task
       ws = run_automate_method(ServiceTemplate.find_by_name("top"),
                                root_service_template_task,
                                FactoryGirl.create(:service))
-      expect(ws.root['include_service']).to be_true
+      expect(ws.root['include_service']).to be_truthy
```

Comment:
I don't know...I mean our code should probably use the truthy/falsey convention so it's great except when it's not.   @matthewd @Fryguy, thoughts?